plugins/betteronline.bwrd.js

```diff
@@ -7,7 +7,7 @@
 */
 
 var uri = new URL(location.href);
-if (uri.pathname.match(/\/?forum\/t\/\d+\/?/)) {
+if (uri.pathname.match(/\/t\/\d+\/?/)) {
```

Comment:
would be nice if the plugin also worked on profiles, just saying

Reply:
Good idea.